index.html

```diff
@@ -1762,6 +1870,96 @@ <h2>
+        <li>[=`url`=]
+        </li>
+      </ul>
+      <p>
+        A user agent SHOULD use these values to register the web application as
+        a handler with the operating system. When the user activates a protocol
+        handler URL, the user agent SHOULD run <a>handling a protocol
+        launch</a>.
+      </p>
+      <p class="note">
+        [[HTML]]'s {{NavigatorContentUtils/registerProtocolHandler()}} allows
+        web sites to register themselves as possible handlers for particular
+        protocols. What constitutes valid <code>protocol</code> and
+        <code>url</code> values for <a>protocol handler description</a>s is
+        defined in that API. Also note that the [[HTML]] API uses
+        [=url/scheme=] where we use <code>protocol</code> but the same
+        restrictions apply.
+      </p>
+      <section>
+        <h3>
+          `protocol` member
+        </h3>
+        <p>
+          The <dfn>protocol</dfn> member of a <a>protocol handler
+          description</a> is a <a>string</a> that represents the protocol to be
+          handled, such as `mailto` or `web+auth`.
+        </p>
+        <p>
+          The <a>protocol</a> member of a <a>protocol handler description</a>
+          is equivalent to
+          {{NavigatorContentUtils/registerProtocolHandler()}}'s |scheme|
+          argument defined in [[HTML]].
+        </p>
+      </section>
+      <section>
+        <h3>
+          `url` member
+        </h3>
+        <p>
+          The <dfn>url</dfn> member of a <a>protocol handler description</a> is
+          the <a>URL</a> [=manifest/within scope=] of the application that
+          opens when the associated protocol is activated.
+        </p>
+        <p>
+          The <a>url</a> member of a <a>protocol handler description</a> is
+          equivalent to {{NavigatorContentUtils/registerProtocolHandler()}}'s
+          <code>url</code> argument defined in [[HTML]].
+        </p>
+      </section>
+      <section>
+        <h3>
+          <dfn>Handling a protocol launch</dfn>
+        </h3>
+        <p>
+          When a <a>protocol handler description</a>
+          <var>protocol_handler</var> having [=manifest=] <var>manifest</var>
+          is invoked, it goes through the same steps used to [=invoke a
+          protocol handler=] defined in [=HTML=], where the user agent SHOULD
+          navigate to [=url=] and the appropriate browsing context is set to a
+          new top level browsing context.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Privacy and Security considerations
+        </h3>
+        <p>
+          Privacy concerns for custom scheme handling are detailed in the
+          Security and privacy section of
+          {{NavigatorContentUtils/registerProtocolHandler()}}.
+        </p>
+        <p>
+          The user agent MUST ask for permission when using a protocol handler
+          for the first time. This feature requires user interaction and a
```

Comment:
I don't understand this second sentence. Are you trying to write a normative requirement that registration requires user interaction? Are you justifying the previous MUST requirement? Please rephrase this.